anoncreds/src/data_types/anoncreds/cred_def.rs

```diff
@@ -26,18 +25,17 @@ impl SignatureType {
     }
 }
 
-#[derive(Debug)]
-#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[derive(Debug, Serialize, Deserialize)]
 pub struct CredentialDefinitionData {
     pub primary: ursa_cl!(CredentialPrimaryPublicKey),
-    #[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
Is Serde not going to be a feature anymore?

Reply:
Ah I see that it came from another macro which was based on a config on whether cl/cl_native was enabled. 

Right now, this macro can be removed and it can be imported directly as we always pull in cl/cl_native right now.

Reply:
Yeah, it was enabled with cl_native feature, but these features are required for anoncreds code, so I removed them (no need for conditional code).